swagger.json

```diff
@@ -5112,6 +5112,12 @@
                     "description": "ID of the Samsara dispatch job.",
                     "example": 773
                 },
+                "vehicleId": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description": "Vehicle ID of the driver at document creation.",
```

Comment:
nit: maybe "VehicleID of the vehicle assigned to the driver at document creation." ?